graylog2-server/pom.xml

```diff
@@ -165,6 +165,11 @@
             <groupId>com.fasterxml.jackson.module</groupId>
             <artifactId>jackson-module-jsonSchema</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.module</groupId>
+            <artifactId>jackson-module-scala_2.13</artifactId>
+            <version>2.9.10</version>
```

Comment:
Why do we hard code the version here? I guess that should use our regular `jackson.version` property?

Reply:
@mpfz0r I guess this is a question for you.

Reply:
This is a workaround for https://github.com/FasterXML/jackson-bom/pull/38
I'll add a comment.

We cannot use our `jackson.version` because that module hasn't been released:

`Could not find artifact com.fasterxml.jackson.module:jackson-module-scala_2.13:pom:2.9.10.20200411 in sonatype-nexus-releases (https://oss.sonatype.org/content/repositories/releases)`